sdk/monitor/monitor-query/src/internal/modelConverters.ts

```diff
@@ -373,6 +385,35 @@ export function convertGeneratedTable(table: GeneratedTable): LogsTable {
       }
 
       return row;
-    })
+    }),
+    columnDescriptors: table.columns
   };
 }
+
+/**
+ * @internal
+ */
+export function convertBatchQueryResponseHelper(response: GeneratedBatchQueryResponse): any {
+  try {
+    const parsedResponseBody: GeneratedBatchQueryResults = JSON.parse(
+      response.body as any
+    ) as GeneratedBatchQueryResults;
+    return {
+      visualization: parsedResponseBody.render,
+      status: "Success",
+      statistics: parsedResponseBody.statistics,
+      error: parsedResponseBody.error,
+      tables: parsedResponseBody.tables?.map((table: GeneratedTable) =>
+        convertGeneratedTable(table)
+      )
+    };
+  } catch (e) {
+    return {
+      visualization: response.body?.render,
+      status: "Success",
```

Comment:
An error occurred, but the status is `Success`?

Reply:
I am updating it in the conditions later. I couldn't put a complex logic in this one line